.github/workflows/nexus-operator-integration-checks.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
           version: latest
-          args: --enable=golint --timeout=2m
+          args: --enable=revive --timeout=2m --skip-files=pkg/test/client.go,pkg/framework/kind/kinds.go,controllers/nexus/resource/validation/defaults.go
```

Comment:
golint was deprecated, hence the change to `revive`.

`fake`, which we use on `pkg/test/client.go` was also deprecated, so I added it to the skip list until a more definitive solution.

The other two files are full of constants and vars which are sufficiently described by their names. Adding comments would render pearls like:

```go
// NexusCommunityImage is the community image of Nexus
NexusCommunityImage = "docker.io/sonatype/nexus3"
```

Not very productive, so I added them to the skip list.